src/muler/igrins.py

```diff
@@ -66,19 +70,29 @@ def __init__(self, *args, file=None, order=10, cached_hdus=None, **kwargs):
                 raise NameError("Cannot identify file as an IGRINS spectrum")
             grating_order = grating_order_offsets[band] + order
 
-            sn_file = file[:-13] + "sn.fits"
+            sn_file = file[19] + "sn.fits"
```

Comment:
This change breaks the existing code to read in the Signal-to-noise file.   The existing code strips the last 13 characters (`"spec_A0V.fits"`) and replaces them with `"sn.fits"`.  This new code only takes the 19th character and joins it with "sn.fits".

Reply:
IGRINS pipeline output file names are 19 characters long plus the suffix (e.g. SDCH_20160131_0122.sn.fits).  It should work unless you are planning on compatiblity with renamed files.

Reply:
I switched back to the old method and added an if ladder to handle both cases for spec_A0V.fits and spec.fits.